_drafts/i2c-overview.md

```diff
@@ -0,0 +1,402 @@
+---
+title: "I2C in a Nutshell"
+description:
+  ""
+tag: [buses]
+author: francois
+image: 
+---
+
+<!-- excerpt start -->
+I2C is perhaps the most commonly used bus to connect ICs together. As such,
+firmwae engineers encounter it on most project. In this post, we explain how I2C
+works and go through common bugs and how to investiage them.
```

Comment:
maybe: the two ands sound a little wordy
> works and go through common bugs and how to investigate them.

works, explore common bugs and investigate how to debug these issues.